tools/crm_mon.c

```diff
@@ -638,6 +686,19 @@ main(int argc, char **argv)
         }
     }
 
+    switch (fence_history_level) {
+        case 3:
+            fence_full_history = TRUE;
+        case 2:
+            show |= mon_show_fence_history;
+        case 1:
+            fence_history = TRUE;
+            fence_connect = TRUE;
+            break;
```

Comment:
It's a good idea to add /* fall through */ comments because that's not always obvious to someone reading the code

Reply:
True I've overlooked such things in other people's code already ...